<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCS</class>
 <widget class="QDialog" name="AddCS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="input_layout_name">
     <item>
      <widget class="QLabel" name="label_cs_name">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="error_cs_name">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_cs_name"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="input_layout_gender">
     <item>
      <widget class="QLabel" name="label_cs_gender">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="error_cs_gender">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="input_cs_gender">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_cs_yob">
     <property name="text">
      <string>Year of birth:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="input_cs_yob">
     <property name="minimum">
      <number>-1000</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="input_cs_alive">
     <property name="text">
      <string>Is this scientist still alive?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="input_layout_yod">
     <item>
      <widget class="QLabel" name="label_cs_yod">
       <property name="text">
        <string>Year of death:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="error_cs_yod">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSpinBox" name="input_cs_yod"/>
   </item>
   <item>
    <widget class="QPushButton" name="button_cs_add">
     <property name="text">
      <string>Add scientist</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
